src/colors/Colors.mdx

```diff
@@ -15,10 +15,23 @@ import {
 
 <ColorPalette>
   <ColorItem title="Collage Red" colors={collage.red} />
-  <ColorItem title="Multicultural Yellow" colors={collage.yellow} />
-  <ColorItem title="Generations Orange" colors={collage.orange} />
-  <ColorItem title="LGBTQ+ Purple" colors={collage.purple} />
-  <ColorItem title="Gender Teal" colors={collage.teal} />
+
+{' '}
```

Comment:
Why these spaces?

Reply:
@Alderr i think Prettier/lint/storybook migration added things for no reason